spec/unit/puppet/provider/mongodb_database/mongodb_spec.rb

```diff
@@ -48,14 +48,14 @@
 
   describe 'create' do
     it 'makes a database' do
-      expect(provider).to receive(:mongo_eval)
+      expect(provider).to have_received(:mongo_eval)
```

Comment:
This is odd to read because I wouldn't expect it to have received `mongo_eval` at this point.

Reply:
@ekohl I am not sure if this is the right solution, but according to the following documentation the tests have passed.: 
* https://relishapp.com/rspec/rspec-mocks/docs/basics/spies#spy-on-a-method-on-a-partial-double